resources/artifact-ignores.properties

```
@@ -234,6 +234,8 @@ tepco-epuw = https://wiki.jenkins-ci.org/display/JENKINS/TEPCO+Electric+Power+Us
 TestExecuter                     # renamed to selected-tests-executor
 # service discontinued
 testflight = https://wiki.jenkins-ci.org/display/JENKINS/Testflight+Plugin
+# service discontinued
+testproject                      # service discontinued
```

```suggestion
testproject  = https://github.com/jenkins-infra/update-center2/pull/696
```

so that current users get a warning

---

All users got an email. But OK.